src/FINDOLOGIC/Objects/JsonResponse.php

```diff
@@ -35,4 +35,17 @@ public function getSuggestions()
     {
         return $this->suggestions;
     }
+
+    /**
+     * Filter the suggestions base on the specified block types.
+     *
+     * @param array $blockTypes The expected block types.
```

Comment:
I do not think that this is the best description for this parameter. Think of something like `Block types that should be filtered`. Maybe you find a better comment describing what this parameter should contain.

Reply:
Added a new comment please have a look if youre fine with it

Reply:
Yes that is a very good comment :)